the-graph/the-graph-menu.js

```diff
@@ -130,13 +138,25 @@ var Menu = React.createFactory( createReactClass({
   displayName: "TheGraphMenu",
   radius: config.radius,
   getInitialState: function() {
+    var initialState = {};
+    var self = this;
+
+    Object.keys(this.props.menu).forEach(
+        function (direction) {
+          var directionObj = self.props.menu[direction];
+          var key = direction + 'tappable';
+
+          // If visibility is set for a given direction, check if the current node should be able to see it.
+          if (!directionObj.isVisible) {
+            initialState[key] = directionObj.action;
+          } else if (directionObj.isVisible(self.props.options)) {
```

Comment:
Not sure if posible, but this would break if `isVisible` gets initialized as `true`.

Reply:
If `isVisible` SHOULD be a function, and that's documented somewhere, then yeah, go for option 1.